public/javascripts/filters.js

```diff
@@ -24,30 +23,21 @@ function showFilters() {
   }); //filters each 
 } //showfilters
 
-function addFilterCloser() {
-  $j('dl.filters').before( $j('<p class="narrow-shown hidden"><button class="close action" aria-label="close">&times;</button></p>') );
-}
-
 function setupMobileFilters() {
   var filters = $j('form.filters');
   var outer = $j('#outer');
-  var show_link = $j('#go_to_filters');
-  var hide_link = $j(filters).find('.close');
+  var filters_link = $j('#go_to_filters');
+  var leave_filters = $j('#leave_filters').find('a');
+  var focusable_item = filters.find(':focusable').first();
 
-  show_link.click(function(e) {
-    e.preventDefault();
-    filters.removeClass('narrow-hidden');
+  filters_link.click(function(e) {
```

Comment:
Identifier 'filters_link' is not in camel case.<br>'e' is defined but never used.